rskj-core/src/test/java/co/rsk/config/ConfigLoaderTest.java

```diff
@@ -181,13 +168,7 @@ public void detectExpectedScalarValueProblemInList() {
 
         mockConfigFactory(defaultConfig, expectedConfig);
 
-        try {
-            loader.getConfig();
-        } catch (RskConfigurationException e) {
-            assertThat(e.getMessage(), containsString(ConfigProblems.expectedScalarValueProblem("expectedKey[0]", EMPTY_LIST_VALUE)));
-
-            throw e;
-        }
+        loader.getConfig();
```

Comment:
with the latest change, this test and the one above it is not making any assertion